canonical-websites/design.ubuntu.com
# remove mobile from site

## Done

* removed /apps section
* removed /scopes section
* removed apps and scopes from nav

## QA

1. see that /apps section is removed
2. see that /scopes section is removed
3. see that Fonts replaces Apps and Scopes in the menu
